src/components/redirect.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import StaticRouter from 'react-router-dom/StaticRouter';
+import Redirect from './Redirect';
+
+const renderToContext = to => {
+	const context = {};
+
+	ReactDOMServer.renderToString(
+		<StaticRouter location={new URL('http://example.com')} context={context}>
+			<Redirect to={to} />
+		</StaticRouter>
+	);
+	return context;
+};
+
+describe('Server rendering', () => {
+	it('sets React Router context for "local" redirect, nothing from Redirect.rewind()', () => {
```

Comment:
sort of. `Redirect.rewind()` is provided by `withSideEffect`, and should only return a value when the underlying React Router `<Redirect>` can't handle the URL - i.e. `rewind()` should only return a value when the Redirect is to an external URL. I'll add another inline comment [here](https://github.com/meetup/meetup-web-platform/pull/289/files/892b6732a97a369d4a6f005d8f7dd98f8af9b22e#diff-d88069df2a21944fc5327a27969009f7R65) to make it more explicit